### PR TITLE
Full review

### PR DESCRIPTION
**Pull request by dnna2610**

### CODE REVIEW

**Comment by dnna2610** [comment]
This should use a linter and a formatter.  I suggest `black` and `pylint`.

File: src/sp_editor/core/_execute.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import os
+import read_excel
```
[the remark this answers is not in the document]
Comment (dnna2610) [reply src/sp_editor/core/_execute.py:2]: seems wrong in multiple places.

File: src/sp_editor/core/spcolumn_cti.py [hunk 2]
```diff
@@ -0,0 +1,1102 @@
...
+            option18,
+            option19,
+            option20,
+            option21,
+            option22,
+            option23,
+            option24,
+            option25,
+            option26,
+            option27,
```
[the remark this answers is not in the document]
Comment (Anhbq1298) [reply src/sp_editor/core/spcolumn_cti.py:198]: Thank you for your comments, this only acts as the placeholder for now. Notice your suggestions also, and will try to figure out the way.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# SP-editor project
+# Empowering Structural Engineers with Seamless Communication Between SpColumn and ETABS
+
+[build-system]
+requires = ["setuptools", "wheel"]  # PEP 508 specifications.
+
+[project]
+name = "sp_editor"
```
[the remark this answers is not in the document]
Comment (tudothanhpt) [reply pyproject.toml:8]: Do we need to change the repository's name to "sp_editor" as well, not sure if it will effect or not

**Pull request merged**